python/mlc_chat/quantization/group_quantization.py

```diff
@@ -207,20 +213,20 @@ def _quantize(  # pylint: disable=too-many-locals
         weight: te.Tensor,
     ) -> Tuple[te.Tensor, te.Tensor]:
         """Group quantization for weight tensor, defined in tensor expression."""
-        assert len(weight.shape) == 2
```

Comment:
In what case is `weight.ndim` not equal to 2?

Reply:
moe expert weight